src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -17,9 +17,12 @@
 global $argv, $argc;
 $argv = isset($_SERVER['argv']) ? $_SERVER['argv'] : [];
 $argc = isset($_SERVER['argc']) ? $_SERVER['argc'] : 0;
-$getEnvVar = function ($name, $default = false) use ($argv) {
-    if (false !== $value = getenv($name)) {
-        return $value;
+$getEnvVar = function ($name, $default = false, $useDefaultWhenEmpty = false) use ($argv) {
+    $value = getenv($name);
+    if (false !== $value) {
+        if ($value || !$useDefaultWhenEmpty) {
+            return $value;
+        }
```

Comment:
the empty check should be explicit IMHO

`if (false !== ($value = getenv($name)) && (!$useDefaultWhenEmpty || '' !== $value)) {`

Reply:
thx for the quick review did update the condition!